tests/tests_common_test.go

```diff
@@ -235,3 +236,59 @@ func expectUserCannot(sars *authv1.SubjectAccessReviewSpec) {
 			sars.ResourceAttributes.Subresource, sars.ResourceAttributes.Name, sars.ResourceAttributes.Group,
 			sars.ResourceAttributes.Version, sars.ResourceAttributes.Namespace))
 }
+
+func GetVMIPodLogs(vmi *v1.VirtualMachineInstance) (string, error) {
```

Comment:
Ok, in this case it is not important to move it.

But for future PRs, there is a guideline in software engineering, that says it is better to write specific code and generalize only when needed. It is not necessary to create generic functions that may be used in the future, because we don't know for certain if they will. When they will be needed, we can easily create them. (I don't remember the name of this guideline)

In this case it is not that important, but please keep it in mind, when you think about future tests that may use it.
If you don't have a specific test in mind that would use it, then it is probably not needed now.